### ISSUE TITLE
Error when running brew install

### ISSUE THREAD

**Issue by soyzamudio** [jackbackes/happycommit#18]
I'm getting an error when I run `brew install jackbackes/homebrew-git-commit-gpt/git-commit-gpt`

Here is the error:

```==> Fetching jackbackes/git-commit-gpt/git-commit-gpt
==> Downloading https://github.com/jackbackes/happycommit/archive/refs/tags/v0.2.0.tar.gz
Already downloaded: /Users/josezamudio/Library/Caches/Homebrew/downloads/d3c3075f30cce1061c5cd8bebe58774f4af85a09f56a83faf0954382ee26c24b--happycommit-0.2.0.tar.gz
==> Installing git-commit-gpt from jackbackes/git-commit-gpt
==> just install
Last 15 lines from /Users/josezamudio/Library/Logs/Homebrew/git-commit-gpt/01.just:
   --> src/main.rs:466:36
    |
466 |     let tokenized_iter = tokenizer.split_by_token_with_special_tokens(code_changes);
    |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `encode_with_special_tokens`

error[E0599]: no method named `split_by_token_with_special_tokens` found for struct `CoreBPE` in the current scope
   --> src/main.rs:472:10
    |
472 |         .split_by_token_with_special_tokens(initial_prompt)
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `encode_with_special_tokens`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `happycommit` due to 2 previous errors
error: failed to compile `happycommit v0.1.0 (/private/tmp/git-commit-gpt-20230425-16952-1xlyork/happycommit-0.2.0)`, intermediate artifacts can be found at `/private/tmp/git-commit-gpt-20230425-16952-1xlyork/happycommit-0.2.0/target`
error: Recipe `install` failed on line 15 with exit code 101

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/jackbackes/homebrew-git-commit-gpt/issues```

**Comment by jackbackes** [comment]
Thanks for the comment - I need to update the Homebrew repo with the most recent version.

**Comment by jackbackes** [comment]
resolved in version 0.2.1. Please try again!
